blogposts/sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph.md

```diff
@@ -0,0 +1,22 @@
+---
+title: SoFi moves fast on hundreds of microservices with Sourcegraph
+author: Vanesa Ortiz
+publishDate: 2019-11-19T10:00-07:00
+tags: [
+  blog
+]
+slug: sofi-moves-fast-on-hundreds-of-microservices-with-sourcegraph
+heroImage: https://about.sourcegraph.com/case-studies/sofi-sourcegraph-case-study.png
```

Comment:
I would consider using one of the iphone with financial info images from the home page, as it's not clear what the woman on the swing symbolizes to me.

Reply:
OK, I'll ask Victoria to update it.